src/ng/directive/select.js

```diff
@@ -650,6 +655,21 @@ var selectDirective = function() {
           return !value || value.length === 0;
         };
 
+      } else {
+
+        var originalIsEmpty = ngModelCtrl.$isEmpty;
+
+        // A single select is also considered empty when the unknown option is present, or when
+        // the empty option is selected
+        ngModelCtrl.$isEmpty = function(value) {
+
+          return originalIsEmpty(value) ||
+            !!selectCtrl.unknownOption.parent().length ||
+            // When the empty option is selected from the scope, then the viewValue can be different
+            // than '', and the originalIsEmpty fn will not catch this case
+            (selectCtrl.hasEmptyOption &&
```

Comment:
Note that this condition only affects empty option required validation, not unknown option - commit message needs to be updated.